src/bundle-cli.ts

```diff
@@ -10,57 +10,70 @@ import { argv } from "./arguments";
 import { Launcher } from "./launcher";
 
 function resolveVerbosity(verbosity: any): number {
-    // Convert given value to an appropriate Verbosity enum value.
-    // 'as any as number' is used because TypeScript thinks
-    //  that we cast string to number, even though we get a number there
-    return (Contracts.Verbosity[verbosity] as any) as number;
+  // Convert given value to an appropriate Verbosity enum value.
+  // 'as any as number' is used because TypeScript thinks
+  //  that we cast string to number, even though we get a number there
+  return (Contracts.Verbosity[verbosity] as any) as number;
 }
 
-function argumentsToConfig(argumentValues: Contracts.ArgumentsValues): Contracts.Config {
-    return {
-        Destination: argumentValues.dest,
-        Entry: argumentValues.entry,
-        DedupeGlobs: argumentValues.dedupe,
-        Verbosity: resolveVerbosity(argumentValues.verbosity),
-        IncludePaths: argumentValues.includePaths,
-        IgnoredImports: argumentValues.ignoredImports,
-        ProjectDirectory: path.resolve(process.cwd(), argumentValues.project)
-    };
+function argumentsToConfig(
+  argumentValues: Contracts.ArgumentsValues
+): Contracts.Config {
+  return {
+    Destination: argumentValues.dest,
+    Entry: argumentValues.entry,
+    DedupeGlobs: argumentValues.dedupe,
+    Verbosity: resolveVerbosity(argumentValues.verbosity),
+    IncludePaths: argumentValues.includePaths,
+    IgnoredImports: argumentValues.ignoredImports,
+    ProjectDirectory: path.resolve(process.cwd(), argumentValues.project)
+  };
 }
 
 async function main(argumentValues: Contracts.ArgumentsValues): Promise<void> {
-    const config = argumentsToConfig(argumentValues);
-    const isWatching = argumentValues.watch != null;
-    const bundler = new Launcher(config);
+  const config = argumentsToConfig(argumentValues);
+  const isWatching = argumentValues.watch !== null;
+  const noWatch = argumentValues.watch === "";
+  const bundler = new Launcher(config);
 
-    if (argumentValues.verbosity !== Contracts.Verbosity.None && (argumentValues.entry == null || argumentValues.dest == null)) {
-        console.error("[Error] 'entry' and 'dest' are required.");
-        process.exit(1);
-    }
+  if (
+    argumentValues.verbosity !== Contracts.Verbosity.None &&
+    (argumentValues.entry == null || argumentValues.dest == null)
+  ) {
+    console.error("[Error] 'entry' and 'dest' are required.");
+    process.exit(1);
+  }
 
-    if (argumentValues.verbosity !== Contracts.Verbosity.None && isWatching && argumentValues.watch === "") {
-        console.error("[Error] 'watch' must be defined.");
-        process.exit(1);
-    }
+  if (

```

Comment:
Adjustments to logic to include noWatch flag here and line 46 - end of bundle-cli.ts